### PR TITLE
1805 cross validation checking for award refs across federal awards and federal awards audit findings workbooks

### PR DESCRIPTION
**Pull request by sambodeme**
## Description
This cross-validation ticket checks that any award reference mentioned in the Federal Awards Audit Findings workbook is present in the Federal Awards workbook. In other words, the collection of award references in the Federal Awards Audit Findings workbook is a subset of the award references in the Federal Awards workbook.
#1805 

To test locally, run:
-`make docker-clean`
-`make docker-first-run`
-`docker compose up`
-`make sure backend-web is running properly and run this cmd in another terminal `docker compose run web python manage.py test audit.cross_validation.test_check_award_ref_declaration`

### CODE REVIEW

File: backend/audit/cross_validation/errors.py [hunk 1]
```diff
@@ -36,12 +36,28 @@ def err_missing_tribal_data_sharing_consent():
...
+def err_award_ref_not_declared(award_refs: list):
+    is_plural = len(award_refs) > 1
+    if is_plural:
+        award_refs_str = (
+            ", ".join(str(ar) for ar in award_refs[:-1]) + " and " + str(award_refs[-1])
+        )
+    else:
+        award_refs_str = str(award_refs[0])
+
+    return (
```
Comment (danswick) [inline backend/audit/cross_validation/errors.py:59]: Really nice error messaging!

**Review by danswick — APPROVED** [review]
Looks good!

Tests all pass locally. I didn't do a complete manual test, mostly because I don't yet have a complete set of test workbooks to upload / get to the cross-validation step.

**Review by jadudm — APPROVED** [review]
Following up on DS's review, and looks good. Merging.

File: backend/audit/cross_validation/__init__.py [hunk 2]
```diff
@@ -55,9 +55,11 @@
     ]
 
 """
-from .number_of_findings import number_of_findings
+from .check_award_ref_declaration import check_award_ref_declaration
+from .check_findings_count_consistency import check_findings_count_consistency
 from .additional_ueis import additional_ueis
 from .auditee_ueis_match import auditee_ueis_match
+from .check_finding_reference_uniqueness import check_finding_reference_uniqueness
 from .award_ref_and_references_uniqueness import award_ref_and_references_uniqueness
```
Comment (jadudm) [inline backend/audit/cross_validation/__init__.py:63]:
```suggestion
```

**Pull request merged**